db/schema.rb

```diff
@@ -3065,110 +3067,6 @@
             ORDER BY alert_generation_runs.school_id, alert_generation_runs.created_at DESC) latest_runs
     WHERE (data.alert_generation_run_id = latest_runs.id);
   SQL
-  create_view "heat_saver_march_2024s", sql_definition: <<-SQL
```

Comment:
I think this has slipped in from your other branch...

Reply:
Ugh. Will fix.